.travis.yml

```diff
@@ -14,7 +14,7 @@ addons:
     - mpich libmpich-dev
     - graphviz flex bison cmake
     - python-numpy python-netcdf4
-    - python3 python3-dev python3-venv python3-pip python3-sphinx
+    - python3 python3-dev python3-venv python3-pip python3-sphinx python3-lxml
```

Comment:
I'm not sure why this is showing up as a diff since it's already on the dev/gfdl branch. https://github.com/NOAA-GFDL/MOM6/blob/dev/gfdl/.travis.yml#L17 . No matter - it's correct.

Reply:
Clicking "Update branch" solved that.